_app/constructor.md

```diff
@@ -3,4 +3,4 @@ header: constructor
 example: new App([attributes], [options])
 ---
 
-Creates an instance of the application, setting the modelUtils, [fetcher](/fetcher), and [router](/router) to the app object.  The attributes are attributes you wish to set on the application.  See the [configuration](#config) object for more details about options that can be passed into the constructor.
+Creates an instance of the application, setting the [modelUtils](/model-utils), [fetcher](/fetcher), and [router](/router) to the app object.  The attributes are attributes you wish to set on the application.  See the [configuration](#config) object for more details about options that can be passed into the constructor.
```

Comment:
`The attributes are attributes you wish to set on the application.` => how about `Pass in the attributes you wish to set on the application.` ?